Applications/Games/The Sims 2/Local/script.js

```diff
@@ -0,0 +1,39 @@
+const LocalScript = include("engines.wine.quick_script.local_installer_script");
+include("engines.wine.verbs.vcrun2010");
+include("engines.wine.verbs.vcrun2013");
+include("utils.functions.net.resource");
+include("utils.functions.filesystem.files");
+include("utils.functions.filesystem.extract");
+include("utils.functions.apps.resources");
```

Comment:
The includes are incorrect

Reply:
There are still incorrect includes. Take a look at the `eslint` report to see what is wrong